hugin/gdocs/google_docs.py

```diff
@@ -0,0 +1,307 @@
+
+import gdata.spreadsheet.service
+import gdata.docs.service
+import gdata
+from gdata import MediaSource
+from gdata import GDataEntry
+import gdata.docs
+from hugin.gdocs import _from_unicode, _to_unicode
```

Comment:
right... this is something I copied and pasted .. I will keep in mind. Do not merge this pull until I do not have a working version. For now this is only to show what I am doing